resources/basics.py

```diff
@@ -496,6 +496,39 @@ def qc_meta_path(data_type: str):
         return 'gs://gnomad/sample_qc/input_meta/gnomad.genomes.streamlined_metadata.2018-10-11.txt.bgz'
 
 
+# liftover data
+def get_gnomad_liftover_data_path(data_type, split=True, version=CURRENT_RELEASE) -> str:
+    """
+    Function to get paths to liftover gnomAD Tables
+    :param str data_type: One of `exomes` or `genomes`
+    :param bool split: Whether the dataset should be split
```

Comment:
`split` isn't used (and I don't think it needs to be - pretty much all tables are assumed to be split).